lib/Conch/Controller/WorkspaceUser.pm

```diff
@@ -89,12 +98,14 @@ sub invite ($c) {
 		$c->log->warn("Email sent to ".$user->email." containing their PLAINTEXT password");
 	}
 
-	Conch::Model::Workspace->new->add_user_to_workspace(
-		$user->id,
-		$c->stash('current_workspace'),
-		$body->{role},
-	);
-	$c->log->info("Add user ".$user->id." to workspace ".$c->stash('current_workspace')->id);
+	# FIXME! do not downgrade a user's existing access to this workspace.
```

Comment:
Do we have a way in the API to change a user's role other than `invite`? If not, `invite` needs to retain the ability to overwrite a user's permissions.

Reply:
(No change needed. Just a comment)